ee/clickhouse/queries/funnels/funnel.py

```diff
@@ -109,7 +120,28 @@ def _format_single_funnel(self, result, with_breakdown=False):
                         "breakdown_value": result[-1],
                     }
                 )
-                # important to not try and modify this value any how - as these are keys for fetching persons
+                # important to not try and modify this value any how - as these
+                # are keys for fetching persons
+
+                # Add in the breakdown to people urls as well
+                # TODO: handle cohorts?
+                converted_people_filter = converted_people_filter.with_data({"funnel_step_breakdown": result[-1]})
+                dropped_people_filter = dropped_people_filter.with_data({"funnel_step_breakdown": result[-1]})
+
+            serialized_result.update(
+                {
+                    "converted_people_url": f"/api/person/funnel/?{urllib.parse.urlencode(converted_people_filter.to_params())}",
```

Comment:
maybe build the absolute url here like for correlations? otherwise the first slash is going to cause issues again

Reply:
have updated 🤞